### PR TITLE
[MM-52996] fix server ci after renaming the project

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
- Fix server CI after renaming the project

#### Ticket Link
MM-52996

#### Related Pull Requests
https://github.com/mattermost/mattermost/pull/23588

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by phoinixgrr — APPROVED** [review]
:shipit:

**Comment by saturninoabril** [comment]
Recommended to cherry-pick into release-7.9 and release-7.8, cc @amyblais

**Review by hmhealey — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -282,8 +282,8 @@ jobs:
       env:
         DOCKER_CLI_EXPERIMENTAL: enabled
       run: |
-        wget https://raw.githubusercontent.com/mattermost/mattermost-server/master/server/build/Dockerfile
-        wget https://raw.githubusercontent.com/mattermost/mattermost-server/master/server/build/entrypoint.sh
+        wget https://raw.githubusercontent.com/mattermost/mattermost/master/server/build/Dockerfile
```
Comment (hmhealey) [inline .github/workflows/ci.yml:285]: I did notice that the old paths here still work fine, but I'm good to change them to be consistent

**Comment by amyblais** [comment]
@saturninoabril Feel free to cherry-pick to release-7.9 and release-7.8 branches.

**Pull request merged**